app/models/document_type.rb

```diff
@@ -20,7 +20,13 @@ def self.all
         hash["contents"] = hash["contents"].to_a.map do |content|
           Field.new(content.merge("label" => I18n.t!("document_types.#{hash['id']}.fields.#{content['id']}.label")))
         end
-        hash["tags"] = hash["tags"].to_a.map(&TagField.method(:new))
+        hash["tags"] = (hash["tags"] || []).map do |tag_config|
```

Comment:
Similarly here. Finding other ways to achieve the lookup should mean we can avoid the need for static helper methods in the later commit.

Reply:
I've now removed the static helper method - didn't like it!